implementations/rust/ockam/ockam_core/src/access_control.rs

```diff
@@ -35,7 +35,7 @@ pub struct AllowAll;
 #[async_trait]
 impl AccessControl for AllowAll {
     async fn is_authorized(&mut self, _local_msg: &LocalMessage) -> Result<bool> {
-        crate::allow()
+        crate::error::allow()
```

Comment:
These shouldn't be errors as they denote a state rather than an error condition.